.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,96 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '43 23 * * *'
```

Comment:
Does this build an image every day, even if no changes have been made?

Reply:
I will bring this in line with the other python workflow, to run on push. Yes the schedule was stupid and I forgot to remove it from here.